addons/mail/static/src/new/views/chatter.xml

```diff
@@ -111,8 +111,25 @@
             <Thread id="thread.id" t-key="thread.id" order="'desc'"/>
         </div>
     </div>
+</t>
 
-
+<t t-name="mail.activity_list" owl="1">
```

Comment:
Not sure how much I like having separate templates that don't have their own component, but ok for now I suppose.

Reply:
this is common in web. a template is basically an inline component without js.